redis/main.lua

```diff
@@ -0,0 +1,35 @@
+local ngx = ngx
+
+local _M = {}
+
+function _M.rewrite()
+    local redis = require "resty.redis"
+    local http = require "resty.http"
+    local red = redis:new()
+    local httpc = http.new()
+    local ok, err = red:connect("127.0.0.1", 6379)
```

Comment:
Could we take the redis address and port as environemnt variables? Also, will it create a redis conenction on each request?